pf2-core/pf2-core.csproj

```diff
@@ -5,4 +5,8 @@
     <RootNamespace>PF2</RootNamespace>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="Core\Rules\Creature\Monster\" />
+    <Folder Include="Core\Rules\Creature\PlayerCharacter\NonPlayerCharacter\" />
```

Comment:
NonPlayerCharacter seems like it should be its own folder inside the Creature Folder, not necessarily a subfolder of PlayerCharacter.  Also, what differentiates NonPlayerCharacter from Monster?

Reply:
I haven't read up on NPC and monster creation rules (might not even be completely out yet) but I know in past editions, there were NPC Classes that a monster might not have. Now a humanoid "monster" might have some NPC or even PC Classes. Again, this is making assumptions based on 1e and not anything I know of from 2e, which is why I stopped there. Doing "NonPcCreature" seems like a clunky name but I'm sure we can come up with something better with some thinking time.

Reply:
Having a unified 'creature' folder makes sense to me.  Splitting NPCs out vs monsters does not.  What, exactly, goes into an NPC block that doesn't fit into a monster block?  And logically speaking, why would that go under the PC folder?

Reply:
Do we think that Creature.Monster.NonPlayerCharacter would make sense for folder structure? I do think that NPC's will have things that "monsters" won't but can't know until the GM guide comes out. I'll put NPC in Monster in Creature for now and we can make a better informed decision later. But I do agree that NPC shouldn't be in PC. :)